ranzen/torch/transforms/cutmix.py

```diff
@@ -31,10 +32,10 @@ def __init__(
         self,
         alpha: float = 1.0,
         *,
-        p: float = 0.5,
+        p: float = 1.0,
         num_classes: int | None = None,
         inplace: bool = False,
-        seed: Optional[int] = None,
+        generator: torch.Generator | None = None,
```

Comment:
This union operator will make it not usable with hydra, right?

Reply:
(on python 3.8)

Reply:
hmm this is true